templates/Includes/ProjectLinks.ss

```diff
@@ -1,5 +1,13 @@
 <% if $CurrentProject %>
 <ul class="project-links">
+	<% if $HasPerm(ADMIN) %>
+		<li>
+			<i class="fa fa-stethoscope"></i>
+			<a target="deploynaut-backend" href="admin/naut/DNProject/EditForm/field/DNProject/item/{$CurrentProject.ID}/ItemEditForm/field/Environments/item/{$CurrentEnvironment.ID}/edit">
+				CMS
```

Comment:
Updated the text. But I want the stethoscope. Most of the time you'd pop over to the CMS to diagnose problems, and it's OPS only feature anyway.